src/main/java/com/yetanalytics/xapi/model/ObjectType.java

```diff
@@ -18,10 +22,17 @@ public enum ObjectType {
         this.displayName = displayName;
     }
 
+    /**
+     * @return Display name of the ObjectType (used in JSON/Serialization)
+     */
     @Override
     @JsonValue
     public String toString() { return displayName; }
 
+    /**
+     * @param name display name to look up correct ObjectType
+     * @return Appropriate ObjectType for the displayName
+     */
```

Comment:
Same here

(On the other hand `toString` is a common enough method that the `@param` annotation by itself might suffice.)